scenarios/modules/loadbalancer/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "scenario" {}
+variable "subnet_id" {}
+variable "loadbalancer_local_ip" {}
```

Comment:
Maybe pass only subnet 3 octets to loadbalancer module? And return local IP if needed?